pipeline/preprocess.py

```diff
@@ -26,52 +45,60 @@ def load_raw(path: str,
         multipage (bool, optional): default=True
             if folder contains stabilized multipage tiffs
             only multipage tiff is supported now
+         pos_dir (bool): "True" if images from each position are saved in separate directories
 
     Returns:
         np.array: numpy array as described above
 
     """
-    fullpath = os.path.join(path, site)
+
     shapes = []
 
     if not multipage:
         print(f"single-page tiffs specified")
         # load singlepage tiffs.  String parse assuming time series and z### format
+        if pos_dir:
+            fullpath = os.path.join(path, site)
```

Comment:
single-page tiffs will contain a position index in the file name like `_p001`.  This approach won't allow a user to specify the position for datasets like that.  Will add this feature in the CLI preproc PR.

Reply:
Thanks for pointing this out! Yes this case was not handled. I just pushed the fix. Feel free to include that in your PR

Reply:
i've included a more precise filter that looks at just the basename and a particular string pattern "p###".   This is in the other PR.